### ISSUE TITLE
Event tracing for Windows support

### ISSUE THREAD

**Issue by MatthiasWeiser** [HdrHistogram/HdrHistogram.NET#33]
It would be nice if there would be an out-of-the-box way to use input from ETW events as input.
I will look into this as I think that things like this should be done out-of-process. As the .NET garbage collector also publishes stats via ETW, it would give a standard way to look at Garbage collection times of programs.

**Comment by LeeCampbell** [comment]
Initial thoughts are that this may be as simple as providing a template C# script to show how you would use an ETW Log file as an input to an HdrHistogram. Presumable the HdrHistogram in this scenario would be used as an intermediary just for reporting purposes?

**Comment by MatthiasWeiser** [comment]
Yes - but I think that in general for real production usage it  should be recommended to log into the ETW and use the etl file for reporting. I have not used [Tx](https://github.com/Microsoft/Tx) yet, but I think that these two libraries could fit together well.
